entries/m/mobile.woolworths.com.au.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Woolworths Mobile": {
```

Comment:
Are the Woolworths and Woolworths Mobile authentication backends different?

Reply:
Yeah, it doesn't seem like I can log into Woolworths Mobile with my Woolworths (supermarket) account. The only way the two are linked is through the [rewards program](https://www.woolworthsrewards.com.au/).